js/pages/samples.js

```diff
@@ -588,6 +614,16 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
+            } else if (site === u_chicago && !location[1]){
+                template += `<div class="row" style="width:100%">
+                <div style="width:100%">
+                    <div class="messagesHeaderFont">
```

Comment:
should there be something inside of the header div?

Reply:
Thanks for correcting me Tony, I removed the empty div.